lib/brain.js

```diff
@@ -638,11 +645,28 @@ Brain.prototype.transitionScreen = function transitionScreen () {
           break
       }
     }
+
+    if (!appState) {
+      // custom info request state
+      machineState = customTier.getState()
+      switch (machineState) {
+        case 'customInfoRequestScreen2':
+          appState = 'customInfoRequestScreen2'
```

Comment:
Could this be changed to a more expressive screen name? Instead of enumerating the possible custom info request screens as '1', '2', '3', etc., having something like 'frontPage', 'numericInput', etc.

Reply:
I'll see if I can make the screen 2 more explicit for each type of input, but the reason is because that's how the wizard organizes info into screen 1 and screen 2.
on the db we also have stuff saved like that:

````JSON
{
   "name":"Birth date",
   "input":{
      "type":"numerical",
      "constraintType":"date"
   },
   "screen1":{
      "text":"On the next screen please enter your birth date.",
      "title":"We need to know your birth date"
   },
   "screen2":{
      "text":"Enter your birth date",
      "title":"Birth date"
   }
}
````